src/SharedElements.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { useState } from 'react';
+import { css, cx } from 'emotion';
+import { textSans, headline, body } from '@guardian/src-foundations/typography';
+import { news, neutral } from '@guardian/src-foundations/palette';
+import { SvgMinus, SvgPlus } from '@guardian/src-icons';
+
+/*
+    Elements that are used on multiple atoms (such as the feedback buttons) 
+    can be imported from here
```

Comment:
I would have these in separate files, perhaps under a `components` folder or something.